### DISCUSSION TITLE
Is there anyway to configure multiple ldap hosts for high availibility

### DISCUSSION THREAD

**Discussion by s-u-b-h-a-k-a-r** [argoproj/argo-cd#5870] [Q&A]
Is there anyway to configure multiple ldap hosts for high availibility using dex ldap connector

connectors:
  - type: ldap
    id: ldap
    name: LDAP
    config:
      host: ''
      insecureSkipVerify: true
      bindDN: ''
      bindPW: 
      usernamePrompt:

**Comment by jannfis** [comment]
I think right there no such possibility exists.

There is an ongoing discussion over at https://github.com/dexidp/dex/issues/1548 about making this happen, and Argo CD is dependent on what Dex provides in this regard.
